src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -0,0 +1,142 @@
+package life.qbic.datamodel.datasets
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+import life.qbic.datamodel.datasets.datastructure.folders.nanopore.*
+
+/**
+ * A dataset that represents a Oxford Nanopore Measurement.
+ *
+ * @author: Sven Fillinger
+ */
+final class OxfordNanoporeMeasurement {
+
+    private final Map<String, DataFolder> folders
+
+    private final List<DataFile> logFiles
+
+    private final MeasurementFolder measurementFolder
+
+    private final boolean pooledMeasurement
```

Comment:
It is more like pooled samples in one measurement. So I would suggest to rename.
```suggestion
    private final boolean pooledSamplesMeasurement
```
DO NOT COMMIT THIS WITHOUT REFACTORING